docs/concepts/flow/executor-args.md

```diff
@@ -44,7 +44,6 @@
 | `metrics_exporter_host` | If tracing is enabled, this hostname will be used to configure the metrics exporter agent. | `string` | `None` |
 | `metrics_exporter_port` | If tracing is enabled, this port will be used to configure the metrics exporter agent. | `number` | `None` |
 | `force_update` | If set, always pull the latest Hub Executor bundle even it exists on local | `boolean` | `False` |
-| `prefer_platform` | The preferred target Docker platform. (e.g. "linux/amd64", "linux/arm64") | `string` | `None` |
```

Comment:
this is wrong

Reply:
The devbot has gone crazy. I have a proper linear branch from the master branch.
```
* c477d6eb98 (HEAD -> introduce-client-prefetch, origin/introduce-client-prefetch) style: fix overload and cli autocomplete
* 8995046c07 test: relax assertion due to different runtimes in ci and local
* 9222fac100 fix: remove unused prefetch arugment from Client args to avoid conflicts
* 8a0d2b3dc7 feat: use optional prefetch before checking class or metadata value
* 722066c54b fix: revert changes to Client constructor
* 900d9e3710 style: fix overload and cli autocomplete
* 4c8cc8d35c feat: add prefetch to client post method
* 6eb4078f68 (origin/master, origin/HEAD, master) docs(contributing/callbacks): fix links and formatting (#5571)
* 72f2eedace feat: use single gateway streamer for CompositeGateway (#5598)
```

Reply:
I don't think there is anything wrong with the devbot. This argument shouldn't exist because prefer_platform does not exist for executors. Try: jina executor --help